### ISSUE TITLE
java.lang.Double cannot be cast to com.facebook.react.bridge.ReadableNativeArray

### ISSUE THREAD

**Issue by rafaelmhp** [tanersener/react-native-ffmpeg#104]
**Description**
When calling RNFFmpeg.getMediaInformation on Android, we're getting a `java.lang.Double cannot be cast to com.facebook.react.bridge.ReadableNativeArray` red screen exception. On Android Studio logcats, I spotted the following exception. 
```
/mobile-ffmpeg: content://com.android.providers.media.documents/document/audio%3A1447: Protocol not found
    Did you mean file:content://com.android.providers.media.documents/document/audio%3A1447?
```
Same code works as expected on iOS. Android side was linked with `react-native link react-native-ffmpeg` as per instructions.

**Expected behavior**
Media information is retrieved. Or at least a handled exception regarding file, media or codec limitations.

**Current behavior**
App crashes

**Screenshots**
N/A

**Logs**
```
I/ReactNativeJS: log level set
I/ReactNativeJS: 'decoded uri', 'content://com.android.providers.media.documents/document/audio%3A1447'
E/unknown:ReactNative: Exception in native call
    java.lang.ClassCastException: java.lang.Double cannot be cast to com.facebook.react.bridge.ReadableNativeArray
        at com.facebook.react.bridge.ReadableNativeArray.getArray(ReadableNativeArray.java:145)
        at com.facebook.react.bridge.ReadableNativeArray.getArray(ReadableNativeArray.java:24)
        at com.facebook.react.bridge.JavaMethodWrapper$6.extractArgument(JavaMethodWrapper.java:82)
        at com.facebook.react.bridge.JavaMethodWrapper$6.extractArgument(JavaMethodWrapper.java:78)
        at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:359)
        at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:158)
        at com.facebook.react.bridge.queue.NativeRunnable.run(Native Method)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:29)
        at android.os.Looper.loop(Looper.java:193)
        at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:232)
        at java.lang.Thread.run(Thread.java:764)
I/ReactNativeJS: [CodePush] Checking for update.
V/FA: Activity resumed, time: 24595599
D/react-native-ffmpeg: Getting media information for content://com.android.providers.media.documents/document/audio%3A1447 with timeout 10000.
D/ReactNative: CatalystInstanceImpl.destroy() start
W/unknown:ViewManagerPropertyUpdater: Could not find generated setter for class com.facebook.react.views.view.ReactViewManager
W/unknown:ReactNative: Invoking JS callback after bridge has been destroyed.
V/FA: Inactivity, disconnecting from the service
I/mobile-ffmpeg: content://com.android.providers.media.documents/document/audio%3A1447: Protocol not found
    Did you mean file:content://com.android.providers.media.documents/document/audio%3A1447?
```

**Environment**
 - React Native Environment Info
```
"react-native": "0.59.4",
"react-native-ffmpeg": "^0.3.4",
```

**Other**
File was picked with `react-native-document-picker`

**Comment by tanersener** [comment]
There are two different issues here. 

The first one is; `RNFFmpeg.getMediaInformation` is called with an unsupported protocol. `content:` protocol is not supported on Android. 

```
I/mobile-ffmpeg: content://com.android.providers.media.documents/document/audio%3A1447: Protocol not found
    Did you mean file:content://com.android.providers.media.documents/document/audio%3A1447?
```

The second issue is; `java.lang.ClassCastException: java.lang.Double cannot be cast to com.facebook.react.bridge.ReadableNativeArray` exception.

According to your logs that exception is received before the `RNFFmpeg.getMediaInformation` call.  So I don't think it comes from `react-native-ffmpeg`. I tested this case locally to make sure of it and didn't receive any crashes. Can you test if you have the same exception without the `RNFFmpeg.getMediaInformation` call?

**Comment by tanersener** [comment]
Apparently there was a bug about `LogLevel` class and that was causing the undefined exception. I fixed that in the new published `0.3.6` release. Can you try using it? It should be working now.

`react-native-ffmpeg` uses file system paths, so you need to convert `content://` to a full path and as far as I know there is not a simple way of doing this. If I were you, I'd manually write data of `content://` into a temp file and use that file with `react-native-ffmpeg`.
